.github/workflows/docker-build-push.yml

```diff
@@ -0,0 +1,51 @@
+name: Docker CI
+
+on:
+  push:
+    branches: [ develop-bridge ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  IMAGE_NAME: xored-mosquitto
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - uses: actions/checkout@v3
+
+    - name: Install xsltproc and docbook-xsl
+      run: |
+        sudo apt update
+        sudo apt install xsltproc docbook-xsl
+
+    - name: Prepare files for the docker build
+      run: make preparedockerbuild
+
+    - name: Set up QEMU
+      id: qemu
+      uses: docker/setup-qemu-action@v1
+      with:
+        image: tonistiigi/binfmt:latest
+        platforms: amd64,arm,arm64
+
+    - name: Set up docker buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Log into registry
+      run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+
+    - name: Build and push docker image
+      run: |
+        IMAGE_ID=ghcr.io/${{ github.repository }}
+        VERSION=latest
```

Comment:
git tags should be supported in order to produce immutable docker image tags

Reply:
fixed